tensorflow/contrib/framework/python/ops/variables_test.py

```diff
@@ -810,5 +810,27 @@ def testInitFromCheckpointWithScopes(self):
       self.assertAllEqual(init_value0, var0.eval())
       self.assertAllEqual(init_value1, var1.eval())
 
+class ZeroInitializerOpTest(tf.test.TestCase):
+
+  def _testZeroInitializer(self, shape, dtype):
+    var0 = tf.Variable(tf.zeros(shape, dtype=dtype))
+    var1 = tf.Variable(tf.ones(shape, dtype=dtype))
+    var0_zero = tf.contrib.framework.zero_initializer(var0)
+    var1_zero = tf.contrib.framework.zero_initializer(var1)
```

Comment:
Indeed, maybe we should make this a RepeatedValueInitializer, with the value specified as an attr, so it can be used to replace both tf.zeros and tf.ones as initializer.